packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -31,9 +31,9 @@ interface ComboboxContentProps {
   readonly children?: ReactElement | ReactElement[];
 
   /**
-   * Placeholder text to display in the search input. Defaults to "Search".
+   * The encapsulating noun for the content of the combobox. Should be pluralized.
```

Comment:
very smol nitpick/suggestion

could we mention where/how it is used? perhaps say "The encapsulating noun for the content of the combobox. Used in the empty state, and search placeholder. Should be pluralized."

the only reason I say that is because I think it's reasonable to assume that the Trigger and Action buttons also use the subject for their text content, but they don't. those are just open ended `label`s

Reply:
Yup, agreed.  Thing about me Zak, is I love a smol nitpick. Bring the nit every time.